.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules/
 /.idea/
+.editorconfig
```

Comment:
well, since the coding style in this repo is 2 spaces. I did the .editorconfig because my editor by default does 4. So maybe you wanna include it? I will  push one to un-ignore it